Support/Scripts/run-lldb-tests.sh

```diff
@@ -130,7 +130,7 @@ else
   export LLDB_DEBUGSERVER_PATH="$build_dir/ds2"
 fi
 
-export LLDB_TEST_TIMEOUT=45m
+export LLDB_TEST_TIMEOUT=8m
```

Comment:
So it seems like an 8m timeout is enough. All the builds passed, and only one timed out, and it's one that's known to cause actual issues.

I'll re-run this a few times to make sure we don't make things worse with this timeout, but I think we should be good.